cedar-policy-core/src/est.rs

```diff
@@ -145,7 +142,6 @@ impl TryFrom<cst::Policy> for Policy {
             resource,
             conditions,
             annot_success,
-            errs.is_empty(),
```

Comment:
I'm not convinced that we actually always have an error here.  `errs` was constructed from a possibly-empty vector.  I'm wondering whether this check should be replaced with a check on whether the `ParseErrors` is `Unknown`, in order to maintain the previous semantics.

Reply:
Yep, realizing that now. Will comment again once I've pushed a fixed version.

Reply:
Ok, new version pushed.

Reply:
The new version here continues to call `.is_empty()`, but `.is_empty()` [will always return false](https://docs.rs/nonempty/latest/nonempty/struct.NonEmpty.html#method.is_empty), so that's not a meaningful check, and in fact might (? not sure) have different behavior than before this PR in an undesirable way, because even if `errs` was created from an empty `Vec`, it will report `.is_empty() == false` due to the `Unknown` error being added.